CodeSchemes/mogadishu_sub_district.json

```diff
@@ -0,0 +1,257 @@
+{
+  "SchemeID": "Scheme-07dfc611",
+  "Name": "Mogadishu Sub-District",
+  "Version": "0.0.0.2",
```

Comment:
I presume 0.0.0.1 wasn't checked in to standards, which is why this comes up as new rather than an edit?

Reply:
Yes (there was a previous version of this scheme which didn't have the code 'kaxda' - I bumped the least significant digit of the version when adding that code)